daml-script/test/upgrades/coin-upgrade-v1-v2/CoinUpgrade.daml

```diff
@@ -69,6 +72,68 @@ create_v2_softFetch_v1 = do
   cid <- alice `submit` createCmd Coin_2_0_0.Coin.Coin with
     issuer = alice
     owner = alice
+    obs = []
   _ <- alice `submitMustFail` createAndExerciseCmd (Aux alice) SoftFetch_Coin_1_0_0 with
     cid = coerceContractId cid
   pure ()
+
+
+template Aux2 -- for softExercise
```

Comment:
I'm okay with adding these helper choices under the existing `Aux`

Reply:
I'm trying to keep related code close.